packages/lib-classifier/src/plugins/tasks/DrawingTask/components/helpers/getIcon.js

```diff
@@ -40,11 +40,12 @@ const icons = {
   polygon: Polygon,
   rectangle: Rectangle,
   rotateRectangle: RotateRectangle,
+  temporalPoint: Point,
```

Comment:
We might consider a different icon, but something that can be followed up later.